frontend/components/mobile/MobileNavbar.tsx

```
@@ -13,27 +13,38 @@ export default function MobileNavbar() {
   const handleSearchExpanded = () => {
     setSearchExpanded(!searchExpanded);
   };
   return (
-    <div className="bg-white fixed top-0 w-screen z-50">
-      <div className="flex items-center py-2 px-4">
-        <Link href="/">
-          <Image src={Logo} alt="Logo" className="w-10 h-10 mr-4" />
+    <header className="bg-white fixed top-0 w-full z-50">
+      <nav
+        className="flex items-center justify-between h-[var(--navbar-height)] px-4 max-w-full"
+        role="navigation"
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<nav>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)